charts/soroban-rpc/templates/soroban-rpc-ingress.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "common.fullname" . }}
+  {{- if .Release.Namespace }}
+  namespace: {{ .Release.Namespace }}
+  {{- end }}
+  labels:
+    app: {{ template "common.fullname" . }}
+    chart: {{ template "common.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: ClusterIP
+  ports:
+    - name: soroban-rpc
+      port: 8000
+      targetPort: 8000
+  selector:
+    app: {{ template "common.fullname" . }}
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ template "common.fullname" . }}
+  {{- if .Release.Namespace }}
+  namespace: {{ .Release.Namespace }}
+  {{- end }}
+  {{- if (.Values.sorobanRpc.ingress).annotations }}
+  annotations:
+    {{- range $key, $value := .Values.sorobanRpc.ingress.annotations }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
+    {{- end }}
+spec:
```

Comment:
do you think the chart should allow configuring the `spec::ingressClassName` here in template, it's not uncommon to have multiple ingress controllers in a cluster and may need to specify non-default here?